erdblick_app/app/view.component.ts

```diff
@@ -123,6 +134,27 @@ export class ErdblickViewComponent implements AfterViewInit {
 
         this.mouseHandler = new ScreenSpaceEventHandler(this.viewer.scene.canvas);
 
+        this.mouseHandler.setInputAction((movement: any) => {
+            const position = movement.position;
+            const cartesian = this.viewer.camera.pickEllipsoid(
+                new Cartesian2(position.x, position.y),
+                this.viewer.scene.globe.ellipsoid
+            );
+            if (defined(cartesian)) {
+                const cartographic = Cartographic.fromCartesian(cartesian);
+                const longitude = CesiumMath.toDegrees(cartographic.longitude);
+                const latitude = CesiumMath.toDegrees(cartographic.latitude);
+                this.parameterService.tileIdsForSourceData = [...Array(16).keys()].map(level => {
```

Comment:
Please move `tileIdsForSourceData` as a subject into the menu service and get rid of the extra `tileIdsReady` subject. Or do you see a reason why not?

Reply:
I probably can do that. It's chiefly an artefact.